agent/agent_test.go

```diff
@@ -63,15 +66,20 @@ func TestAgentSuite(t *testing.T) {
 	suite.Run(t, new(AgentSuite))
 }
 
+// kim: TODO: fix merge conflict due to task working directory consolidation
```

Comment:
Still have a TODO here.

Reply:
This was waiting on #7030 to be merged, so the merge conflict is gone now.